storagenode/trust/file_source.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package trust
+
+import (
+	"context"
+	"net/url"
+)
+
+// FileSource represents a trust source contained in a file on disk
+type FileSource struct {
+	url  string
+	path string
+}
+
+// NewFileSource creates a new FileSource that loads a trust list from the
+// provide file:// URL. The URL must have either an empty source or be set to
+// localhost. Furthermore, it must not contain user info, query values, or a
+// fragment. The path must be non-empty.
+func NewFileSource(fileURL string) (*FileSource, error) {
+	u, err := url.Parse(fileURL)
```

Comment:
Filepaths are not URLs. Let's not pretend that they are. We already have fixes in place for zap handling them wrong.

https://play.golang.org/p/z7sP3hOyCLF

Reply:
I would make `NewFileSource` take a path as argument.